Makefile.am

```diff
@@ -232,8 +232,13 @@ multi-word: po/cockpit.pot
 title2sentence: multi-word
 	tools/title2sentence.py -i titles -o t2s.sh
 
-include po/Makefile.am
+include containers/Makefile.am
+include containers/flatpak/Makefile.am
```

Comment:
This has nothing to do with moving src/systemd, but then I realized you probably just sorted it. Any chance of doing the sorting as a separate commit?

Reply:
ok.